src/views/EventFilter.js

```diff
@@ -33,16 +33,22 @@ export class EventFilter extends Component {
     _hideDateTimePicker = () => this.setState({ isDateTimePickerVisible: false });
 
     _handleDatePicked = (date) => {
+        _date = Moment(date);
```

Comment:
If you don't put `let/const` before declarations, they're global!

Reply:
😆

Reply:
Ohhhh javascript 😅